### PR TITLE
DOC: Update fromnumeric.py

### PR DESCRIPTION
**Pull request by bhavyaalekhya**
DOC: Add examples for any for positive inifinity and exploring parameters out and keepdims for multidimensional arrays

### CODE REVIEW

File: numpy/core/fromnumeric.py [hunk 1]
```diff
@@ -2406,6 +2409,20 @@ def any(a, axis=None, out=None, keepdims=np._NoValue, *, where=np._NoValue):
...
+    (2,2,3)
+    >>> ex = np.any([[[1,2,4],[3,5,6]],[[3,4,5],[7,8,9]]],axis=2,keepdims=True)
+    >>> ex
+    array([[[ True],
+        [ True]],
+       [[ True],
+        [ True]]])
+    >>> ex.shape
+    (2,2,1)
+
```
Comment (Carreau) [inline numpy/core/fromnumeric.py:2425]: Do we care about the formatting of example in Docs, for example space after commas in parameters ?

```suggestion
    >>> np.any([[1.0,3.5,-0.9],[4.5,7.9,0],[-5.6,11,3.7]], out=np.array(float))
    array(1.0, dtype=object)
    
    >>> np.shape([[[1,2,4],[3,5,6]],[[3,4,5],[7,8,9]]])
    (2,2,3)
    >>> ex = np.any([[[1,2,4],[3,5,6]],[[3,4,5],[7,8,9]]], axis=2, keepdims=True)
    >>> ex
    array([[[ True],
        [ True]],
       [[ True],
        [ True]]])
    >>> ex.shape
    (2,2,1)

```

I'm not a core dev, so it's a real question but I personally find that with spaces (and having consistent code), it's easier to read.

Comment (mattip) [reply numpy/core/fromnumeric.py:2425]: > Do we care about the formatting of example in Docs

yes

**Comment by mattip** [comment]
@bhavyaalekhya will you be returning to this? In addition to the review comments, please add a line of descriptive text above each example to explain what the example is showing. That will give a hint as to why the new examples are needed.

**Review by bhavyaalekhya — COMMENTED** [review]
Added description for the changes made

**Comment by mdhaber** [comment]
There were significant merge conflicts, so I restored `core/fromnumeric.py` from `main` and used some of the ideas from this PR to modify one of the existing examples in the documentation of `any` in `_core/fromnumeric.py`. I reduced the scope to avoid the need for additional iteration. I'll merge once CI finishes. Thanks @bhavyaalekhya!

**Pull request merged**